hack/jenkins/e2e.sh

```diff
@@ -220,6 +222,13 @@ if [[ "${E2E_TEST,,}" == "true" ]]; then
     go run ./hack/e2e.go ${E2E_OPT} -v --test --test_args="${GINKGO_TEST_ARGS}--ginkgo.noColor" || true
 fi
 
+# TODO(zml): We have a bunch of legacy Jenkins configs that are
+# expecting junit.xml to be in ${WORKSPACE} root and it's Friday
+# afternoon, so just put the junit report where it's expected.
+if [ -e ${ARTIFACTS}/junit.xml ]; then
```

Comment:
It's almost like there were junit*.xml patterns sitting right in front of my eyes.